doc/development/rfc/version_numbering.md

```diff
@@ -0,0 +1,146 @@
+# RFC: Version Numbering
+
+Author: Vaclav Petras
+
+Status: Draft
+
+## Summary
+
+Use version number in a format _major.minor.micro_ where
+- _major_ is for large additions or backwards incompatible changes,
+- _minor_ is for additions, and
+- _micro_ is for fixes.
+These numbers are assigned to releases. Development versions, pre-release,
+and builds are marked by additional labels as suffixes.
+All major and minor version numbers are used for releases,
+i.e., there is no distinction between even and odd numbers.
+
+## Background
+
+The increasing versions was always following the same ideas as in
+the Semantic Versioning, i.e., _major_, _minor_, and _patch_
+with major for breaking changes, minor for backwards compatible
+features, and patch for fixes.
+
+For the versions 7.0 till 8.2, there was a distinction between even and odd minor versions.
+Even minor versions were released, while odd minor version marked development version.
+This odd versions never had a patch number assigned and were never released.
```

Comment:
Note: The history to this was the adoption of the Linux kernel versioning system:

https://en.wikipedia.org/wiki/Software_versioning#Odd-numbered_versions_for_development_releases

Indeed, the Linux kernel project abandoned it time ago.

Reply:
Any idea when it was first applied to GRASS GIS? Just to have the history complete there.

Also interesting: Linux kernel moved from 2 to 3 _not_ because of a breaking updates. What are the rules now, I don't know.

Reply:
Here a relevant snippet (not fully history):
https://lists.osgeo.org/pipermail/grass-dev/2007-August/032705.html

Reply:
Thanks. Great material. I updated the document based on these two although I don't reference them in the text.